ephemeralkey.go

```diff
@@ -0,0 +1,42 @@
+package stripe
+
+import "encoding/json"
+
+// EphemeralKeyParams is the set of parameters that can be used when creating
+// an ephemeral key.
+// For more details see https://stripe.com/docs/api#ephemeral_keys.
+type EphemeralKeyParams struct {
+	Params
+	Customer string
+}
+
+// EphemeralKey is the resource representing a Stripe ephemeral key.
+// For more details see https://stripe.com/docs/api#ephemeral_keys.
+type EphemeralKey struct {
+	ID      string `json:"id"`
+	Created int64  `json:"created"`
+	Expires int64  `json:"expires"`
+	Live    bool   `json:"livemode"`
+    AssociatedObjects []struct {
+        ID   string `json:"id"`
+        Type string `json:"type"`
+    } `json:"associated_objects"`
+    RawJSON []byte
+}
+
+// UnmarshalJSON handles deserialization of an EphemeralKey.
+// This custom unmarshaling is needed because the resulting
+// property may be an id or the full struct if it was expanded.
+func (e *EphemeralKey) UnmarshalJSON(data []byte) error {
```

Comment:
So the top comment was copy/paste incorrect, but I think we still need this because of the `RawJSON` field (which is needed to pass back an unparseable EphemeralKey to the frontend - which could happen due to incompatible api versions).